source/reference/minio-server/settings/core.rst

```diff
@@ -295,3 +295,48 @@ Defaults to ``"text/*, application/json, application/xml, binary/octet-stream"``
    Some	types of files cannot be significantly reduced in size.
    MinIO will *not* compress these, even if specified in an :mc-conf:`~compression.mime_types` argument.
    See :ref:`Excluded types <minio-data-compression-excluded-types>` for details.
+
+Erasure Stripe Size
+~~~~~~~~~~~~~~~~~~~
+
+.. tab-set:: 
+
+   .. tab-item:: Environment Variable
+      :sync: envvar
+
+      .. envvar:: MINIO_ERASURE_SET_DRIVE_COUNT
+
+   .. tab-item:: Configuration Variable
+      :sync: config
+
+      .. include:: /includes/common-mc-admin-config.rst
+         :start-after: start-minio-settings-no-config-option
+         :end-before: end-minio-settings-no-config-option
+
+*Optional*
+
+The :ref:`erasure set size <minio-ec-basics>` to apply for all drives in a given :term:`server pool`.
+
+You **must** set this value before you first start the server pool nodes.
+The stripe size selected for the first server pool becomes a baseline minimum for any additional server pools.
+
+You **cannot** change the erasure set stripe size after the cluster has been initialized.
+
+**Do not** deploy custom stripe sizes into production without first validating in lower environments using a mix of performance, stress, and real-workload tests.
+
+.. important::
+
+   Stripe size impacts multiple areas of functionality, including:
+
+   - Storage Capacity
+   - Failure Tolerance
+   - Parity Configuration
+   - Workload Efficiency
+
+   MinIO's default stripe selection algorithms are typically sufficient for the majority of workloads.
```

Comment:
```suggestion
   MinIO's stripe selection algorithms set appropriate defaults for the majority of workloads.
   Changing the stripe size from this default is unusual and generally not necessary or advised.
```

Reply:
Maybe?

Reply:
I like this, it's more assertive, much less wiggle room for people to say "well, I'm special, not typical".  Much harder for someone to believe, "well, I'm inappropriate, not appropriate."